README.md

```diff
@@ -69,7 +69,9 @@ Cats:
 
 ## I'm writing a performance-sensitive application<a id="sec-2-2"></a>
 
-Lean toward Cats, it tends to be faster in aggregate. Are you using a database? Consider [Doobie](https://github.com/tpolecat/doobie), which uses Cats.
```

Comment:
I'm not sure what doobie has to do with performance sensitive here... it is also compatible with either thanks to shims.